shared/chat/conversation/normal/index.desktop.js

```diff
@@ -81,6 +82,21 @@ class Conversation extends React.PureComponent<Props, State> {
     const fileList = e.dataTransfer.files
     const paths = fileList.length ? Array.prototype.map.call(fileList, f => f.path) : []
     if (paths) {
+      for (let path of paths) {
+        // Check if any file is a directory and bail out if not
+        try {
+          // We do this synchronously
+          // in testing, this is instantaneous
+          // even when dragging many files
+          const stat = fs.lstatSync(path)
+          if (stat.isDirectory()) {
+            // TODO show a red error banner on failure: https://zpl.io/2jlkMLm
+            this.setState({showDropOverlay: false})
+            return
+          }
+          // delegate to handler for any errors
+        } catch (e) {}
```

Comment:
i'm always wary of just dropping errors...perhaps log/warn them?

Reply:
Do you think logging these would expose file paths / do we care?

Reply:
hmm good point. Is there a particular error/class of error we're expecting here? Perhaps check for those and let unexpected errors bubble up? I'm just wary of dropping syntax errors and the like.

Reply:
There aren't really any expected since we weren't doing this check before. the docs say it'll be one of [these](https://nodejs.org/api/errors.html#errors_common_system_errors) (permissions and such). I'll change this to log `e.code` so we can get something in case this does pop up.